nservicebus/operations/performance-counters_counters_core_[6].partial.md

```diff
@@ -0,0 +1,7 @@
+include: performance-counters-criticaltime
+
+include: performance-counters-note
+
+include: performance-counters-slaviolation
```

Comment:
Or actually i will alter a link to only refer to the main page for performance counters (which is not ideal, but better than removing the link altogether).